Airports/EGNM/SMR/Geo.txt

```diff
@@ -19,4 +19,72 @@ N053.51.52.531 W001.39.46.105 N053.51.51.690 W001.39.44.825 standHold ;L2
 N053.52.03.167 W001.39.33.443 N053.52.02.409 W001.39.33.459 standHold ;N1
 N053.52.02.947 W001.39.19.788 N053.52.02.206 W001.39.19.790 standHold ;N2
 N053.52.02.809 W001.39.12.085 N053.52.02.071 W001.39.12.109 standHold ;N3
-N053.52.04.088 W001.39.08.366 N053.52.04.061 W001.39.07.078 standHold ;N4
\ No newline at end of file
+N053.52.04.088 W001.39.08.366 N053.52.04.061 W001.39.07.078 standHold ;N4
+
+
```

Comment:
```suggestion
;Tug Release Points
```
Either a comment like this, and maybe one for the hold points, or just remove the extra blank line, upto you.